examples/quick-start-jaeger-nodejs/tracetest/collector.config.yaml

```
@@ -15,10 +15,16 @@ exporters:
     endpoint: jaeger:14250
     tls:
       insecure: true
+  otlphttp:
+    endpoint: http://host.docker.internal:4317
```

For http, the correct port is `4318`:

```suggestion
    endpoint: http://host.docker.internal:4318
```